circle/circle.go

```diff
@@ -18,6 +18,15 @@ Square: 201.06192982974676
 func main() {
 	fmt.Println("The program prints the perimeter and the square of a circle given the radius.")
 	// ++ Your code here! ++
+	fmt.Println("Enter the radius:")
+	var rds float64
```

Comment:
optional: feel free to use just `r` for cases like this one. `r` is an idiomatic name for radiuses, so everyone would understand what's that

Reply:
The same stands for `p` (perimeter) and `a` (area)